src/java.desktop/share/classes/javax/swing/border/EmptyBorder.java

```diff
@@ -102,6 +104,8 @@ public void paintBorder(Component c, Graphics g, int x, int y, int width, int he
      * Reinitialize the insets parameter with this Border's current Insets.
      * @param c the component for which this border insets value applies
      * @param insets the object to be reinitialized
+     * @throws {@code NullPointerException} if {@code insets}
+     *          is {@code null}
```

Comment:
```suggestion
     * @throws NullPointerException {@inheritDoc}
```
I suggest using `{@inheritDoc}` to copy the exception description from the super class for both `EmptyBorder` and `MatteBorder`. (It cannot be used for constructors though.)